### DISCUSSION TITLE
How can I connect NERs to a PERSON if they are related to each other?

### DISCUSSION THREAD

**Discussion by Steven-Data** [explosion/spaCy#9327] [Help: Coding & Implementations] [answered]
I was wondering if there is a way to connect NERs to a PERSON if they are related to each other? For example, there are several NER were extracted by SpaCy, which are Steven (person), Jackson (person), San Jose (city), Google (organization), and 4243214567 (number). I would like to connect San Jose and 4243214567 to Steven because he lives in the city and the number is his phone number, and connect Google to Jackson because he work at the company. Do you have any suggestions for me on this use case? Thanks.

**Comment by polm** [comment, marked as answer]
Have you seen the [relation extraction tutorial](https://github.com/explosion/projects/tree/v3/tutorials/rel_component)? 

It sounds like you might be doing something a little more general/vague than relation extraction, in which case it depends on 1. how you can tell that things are related 2. what kind of output you want from that. A simple way to group things is to put all things in the same sentence in a dict, for example, but there's many ways to do it.
